pkgs/by-name/fa/faircamp/package.nix

```diff
@@ -17,14 +17,14 @@
 
 rustPlatform.buildRustPackage rec {
   pname = "faircamp";
-  version = "1.0.0";
+  version = "1.1.1";
 
   src = fetchFromGitea {
     domain = "codeberg.org";
     owner = "simonrepp";
     repo = "faircamp";
     rev = version;
-    hash = "sha256-Pdt3HjjM4lR6CW3T10eMCvZWV29rbspGKHV7yEUA2U0=";
+    hash = "sha256-+xkCRbbRXsG1ZOS8VVj0gi60kxcM8LmNRx1XeJ7Xtlw=";
   };
 
   cargoLock = {
```

Comment:
Could you use fetchCargoVendor? See https://github.com/NixOS/nixpkgs/pull/349360